src/status_im2/contexts/shell/view.cljs

```diff
@@ -137,16 +138,20 @@
   []
   [:f>
    (fn []
-     (let [shared-values (animation/calculate-shared-values)]
+     (let [shared-values (animation/calculate-shared-values)
+           show-shell?   (reagent/atom false)]
```

Comment:
Hi, we already have functionality to set default selected tab.
Its added to create-only-event, but not sure why that event didn't got called in mentioned issue.
If it follows some other flow or even, please add this change there too
https://github.com/status-im/status-mobile/blob/ac2d10bc5d2001df8a45be1834dc81d3dcbe2ceb/src/status_im/multiaccounts/login/core.cljs#L522